### ISSUE TITLE
what folder does the rom go into

### ISSUE THREAD

**Issue by SlippingGitty** [longhorn8x/SM64Macprep#1]
<img width="1272" alt="image" src="https://github.com/longhorn8x/SM64Macprep/assets/76500838/4452cc94-190c-480e-b6f1-764442d9e594">

**Comment by longhorn8x** [comment]
hey sorry for mistake on my part you will need sm64ex or render96 or etc, grab the sh to the and put it on that folder sorry for the mistake

**Comment by longhorn8x** [comment]
please let me know if there any problem

**Comment by SlippingGitty** [comment]
oh thanks i'm a dummy lol, i'll do that when i get home and let you know

**Comment by SlippingGitty** [comment]
```
>>>[a little after pressing enter]<<<
.....
extracting textures/water/jrb_textures.00000.rgba16.png
extracting textures/water/jrb_textures.00800.rgba16.png
extracting textures/water/jrb_textures.01800.rgba16.png
extracting textures/water/jrb_textures.02800.rgba16.png
extracting textures/water/jrb_textures.03800.rgba16.png
extracting textures/water/jrb_textures.04800.rgba16.png
extracting textures/water/jrb_textures.05800.rgba16.png
extracting textures/water/jrb_textures.06000.rgba16.png
extracting textures/water/jrb_textures.06800.rgba16.png
extracting textures/water/jrb_textures.07800.rgba16.png
extracting textures/water/jrb_textures.08800.rgba16.png
extracting textures/water/jrb_textures.09000.rgba16.png
extracting textures/water/jrb_textures.0A000.rgba16.png
extracting textures/water/jrb_textures.0A800.rgba16.png
extracting textures/water/jrb_textures.0B800.rgba16.png
gmake[1]: Entering directory '/Users/vozy/64/Render96ex/tools'
gcc-13 -I../include -I. -Wall -Wextra -Wno-unused-parameter -pedantic -std=c99 -O2 -s  aiff_extract_codebook.c -o aiff_extract_codebook -lm
gcc-13 -I../include -I. -Wall -Wextra -Wno-unused-parameter -pedantic -std=c99 -O2 -s -Wno-unused-result -Wno-uninitialized -Wno-sign-compare -Wno-absolute-value sdk-tools/adpcm/vadpcm_enc.c sdk-tools/adpcm/vpredictor.c sdk-tools/adpcm/quant.c sdk-tools/adpcm/util.c sdk-tools/adpcm/vencode.c -o vadpcm_enc -lm
gcc-13 -I../include -I. -Wall -Wextra -Wno-unused-parameter -pedantic -std=c99 -O2 -s -Iaudiofile -Wno-uninitialized sdk-tools/tabledesign/codebook.c sdk-tools/tabledesign/estimate.c sdk-tools/tabledesign/print.c sdk-tools/tabledesign/tabledesign.c -o tabledesign -lm -Laudiofile -laudiofile -lstdc++ -lm
gmake[2]: Entering directory '/Users/vozy/64/Render96ex/tools/audiofile'
g++-13 -std=c++11 -DNDEBUG -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -O2 -I. -c audiofile.cpp
aiff_extract_codebook.c: In function 'fail_parse':
aiff_extract_codebook.c:48:16: warning: implicit declaration of function 'vsnprintf' [-Wimplicit-function-declaration]
   48 |     int size = vsnprintf(NULL, 0, fmt, ap);
      |                ^~~~~~~~~
aiff_extract_codebook.c:12:1: note: include '<stdio.h>' or provide a declaration of 'vsnprintf'
   11 | #include <stdarg.h>
  +++ |+#include <stdio.h>
   12 |
aiff_extract_codebook.c:48:32: warning: 'vsnprintf' argument 2 type is 'in ' where 'long unsigned int' is expected in a call to built-in function declared without prototype [-Wbuiltin-declaration-mismatch]
   48 |     int size = vsnprintf(NULL, 0, fmt, ap);
      |                                ^
<built-in>: note: built-in 'vsnprintf' declared here
aiff_extract_codebook.c:55:41: warning: 'vsnprintf' argument 2 type is 'in ' where 'long unsigned int' is expected in a call to built-in function declared without prototype [-Wbuiltin-declaration-mismatch]
   55 |             size = vsnprintf(formatted, size, fmt, ap);
      |                                         ^~~~
<built-in>: note: built-in 'vsnprintf' declared here
audiofile.cpp:3564:47: warning: 'template<class _Arg, class _Result> struct std::unary_function' is deprecated [-Wdeprecated-declarations]
 3564 |         struct signedToUnsigned : public std::unary_function<SignedType, UnsignedType>
      |                                               ^~~~~~~~~~~~~~
In file included from /opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/string:49,
                 from audiofile.cpp:1735:
/opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/bits/stl_function.h:117:12: note: declared here
  117 |     struct unary_function
      |            ^~~~~~~~~~~~~~
audiofile.cpp:3569:47: warning: 'template<class _Arg, class _Result> struct std::unary_function' is deprecated [-Wdeprecated-declarations]
 3569 |         struct unsignedToSigned : public std::unary_function<SignedType, UnsignedType>
      |                                               ^~~~~~~~~~~~~~
/opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/bits/stl_function.h:117:12: note: declared here
  117 |     struct unary_function
      |            ^~~~~~~~~~~~~~
audiofile.cpp:3762:33: warning: 'template<class _Arg, class _Result> struct std::unary_function' is deprecated [-Wdeprecated-declarations]
 3762 | struct intToFloat : public std::unary_function<Arg, Result>
      |                                 ^~~~~~~~~~~~~~
/opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/bits/stl_function.h:117:12: note: declared here
  117 |     struct unary_function
      |            ^~~~~~~~~~~~~~
audiofile.cpp:3828:29: warning: 'template<class _Arg, class _Result> struct std::unary_function' is deprecated [-Wdeprecated-declarations]
 3828 | struct lshift : public std::unary_function<Arg, Result>
      |                             ^~~~~~~~~~~~~~
/opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/bits/stl_function.h:117:12: note: declared here
  117 |     struct unary_function
      |            ^~~~~~~~~~~~~~
audiofile.cpp:3834:29: warning: 'template<class _Arg, class _Result> struct std::unary_function' is deprecated [-Wdeprecated-declarations]
 3834 | struct rshift : public std::unary_function<Arg, Result>
      |                             ^~~~~~~~~~~~~~
/opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/bits/stl_function.h:117:12: note: declared here
  117 |     struct unary_function
      |            ^~~~~~~~~~~~~~
audiofile.cpp:3930:35: warning: 'template<class _Arg, class _Result> struct std::unary_function' is deprecated [-Wdeprecated-declarations]
 3930 | struct floatToFloat : public std::unary_function<Arg, Result>
      |                                   ^~~~~~~~~~~~~~
/opt/homebrew/Cellar/gcc/13.2.0/include/c++/13/bits/stl_function.h:117:12: note: declared here
  117 |     struct unary_function
      |            ^~~~~~~~~~~~~~
ld: warning: -s is obsolete
ld: warning: -s is obsolete
ld: warning: -s is obsolete
ld: library 'audiofile' not found
collect2: error: ld returned 1 exit status
gmake[1]: *** [Makefile:47: tabledesign] Error 1
gmake[1]: *** Waiting for unfinished jobs....
ar rcs libaudiofile.a audiofile.o
gmake[2]: Leaving directory '/Users/vozy/64/Render96ex/tools/audiofile'
gmake[1]: Leaving directory '/Users/vozy/64/Render96ex/tools'
Makefile:236: *** Failed to build tools.  Stop.
DONE
```

this is the error i received. this is on an M2 mac, if that matters. render64ex

edit: 64ex ran fine

**Comment by SlippingGitty** [comment]
looks like 64 ex ran fine, no idea what or where it outputted tho
